README.md

```diff
@@ -27,6 +28,42 @@ This repository contains the code of the backend used by the [web](https://githu
 
 ---
 
+## What is this?
+
+This is the backend that supports the [italia-app](https://github.com/teamdigitale/italia-app)
+mobile application.
+
+This project is part of the Italian Digital Citizenship initiative, see the
+[main repository](https://github.com/teamdigitale/digital-citizenship) for further information.
```

Comment:
information(s) ?

Reply:
information is uncountable, there is no plural form of it